apps/test/util/runTests.js

```diff
@@ -77,7 +77,8 @@ exec(command, function (err, stdout, stderr) {
     transform: 'ejsify',
     'web-security': false
   })
- .on('error', function () {
+ .on('error', function (e) {
+   console.log(e.stack);
```

Comment:
trace is nonstandard for the web, but is I believe generally preferred in [nodejs](https://nodejs.org/api/console.html#console_console_trace_message) land.

LGTM with either method, though.